_data/toc/software-update-guide.yml

```diff
@@ -123,6 +123,10 @@ pages:
           url: /comp-mgr/install-extensions/b2b-installation.html
           exclude_versions: ["2.1"]
 
+        - label: Manage Inventory Management modules
```

Comment:
I would put this item before B2B, since everyone installs MSI.

Reply:
Swapped